saos-search/src/main/java/pl/edu/icm/saos/search/search/service/RangeFacetToQueryApplier.java

```diff
@@ -29,6 +35,9 @@ public void applyRangeFacet(SolrQuery query, String fieldName, String start, Str
         String fieldParamPrefix = "f." + fieldName + ".";
         
         query.add(FacetParams.FACET_RANGE, fieldName);
+        if (!StringUtils.isBlank(fieldValuePrefix)) {
+            query.add(FacetParams.FACET_PREFIX, fieldValuePrefixAdder.prefixWithSeparator(fieldValuePrefix));
```

Comment:
Facet prefix doesn't have sense in facet of type range. Solr will just skip this parameter. It would be better if you get rid of `fieldValuePrefix` var in this method